scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1339,6 +1475,71 @@ function Update-ConfigYaml([ScriptBlock] $UpdateFunc)
     $configurationYaml | Set-Content $yamlPath -Force
 }
 
+function Validate-GatewaySettings {
+    $certFilesProvided = $false
+    if ($DeviceCACertificate -or $DeviceCAPrivateKey -or $DeviceTrustbundle) {
+        if (-Not (Test-Path -Path $DeviceCACertificate)) {
+            throw "Device CA certificate file $DeviceCACertificate not found. When configuring device certificates, a certificate file is required."
+        }
+        if (-Not (Test-Path -Path $DeviceCAPrivateKey)) {
+            throw "Device CA private key file $DeviceCAPrivateKey not found. When configuring device certificates, a private key file is required."
+        }
+        if (-Not (Test-Path -Path $DeviceTrustbundle)) {
+            throw "Device trustbundle file $DeviceTrustbundle not found. When configuring device certificates, a trust bundle file is required."
+        }
+        $certFilesProvided = $true
+    }
+
+    return $certFilesProvided
+}
+
+function Get-DpsProvisioningSettings {
+    $idCertFilesProvided = $false
+    $attestationMethod="tpm" # default
```

Comment:
Single quotes for strings without interpolation. Spaces around `=`

Reply:
Fixed, thanks.